keps/sig-storage/0033-20181111-transfer-pvc-between-namespaces.md

```diff
@@ -0,0 +1,179 @@
+---
+kep-number: 33
+title: Add ability to transfer PVC between Namespaces
+authors:
+  - "@j-griffith"
+owning-sig: sig-storage
+participating-sigs:
+  - sig-architecture
+reviewers:
+  - TBD
+approvers:
+  - TBD
```

Comment:
Can you please add the SIG Storage chairs/tech leads as approvers?

Reply:
Done